docs/source/markdown/podman-pod-ps.1.md.in

```diff
@@ -86,7 +86,8 @@ Valid placeholders for the Go template are listed below:
 | .Created            | Creation time of pod                                 |
 | .ID                 | Container ID                                         |
 | .InfraID            | Pod infra container ID                               |
-| .Labels             | All the labels assigned to the pod                   |
+| .Label STRING       | Specified label of the pod                           |
```

Comment:
This huge PR, just for this....

Reply:
should we use `<STRING>` or `*string*` as syntax to better display that this is a mandatory argument?

Reply:
That's what I was going for with CAPS, but yours seems more consistent with our conventions.

Angle brackets, probably not, because they'd need to be escaped and that's too hard to enforce. Asterisks (italics) show up only in generated HTML, not in man pages (at least not on my laptop, AFAICT). See attached image; the italics don't show up well in *CAPS*, but they are prominent in *lowercase*.

![recycle](https://github.com/containers/podman/assets/265639/02e7921c-eb55-45f0-851c-4d86464bd811)


If we can find a way to highlight `*italics*` in the terminal man page view, I'd like to go with that approach. Otherwise, slight preference for `CAPS`.

Reply:
I am fine with CAPS

Reply:
Followup: `go-md2man` seems to handle italics-in-tables just fine, and produces what looks like sane \*roff:
```
\&.Recycle \fIpath\fP \fIbool\fP    [unimplemented]
```
...but `man -l` is not respecting that.

Italics and bold *are* possible in tables -- see `systemd.unit(5)` -- I just can't figure out what special magic is needed, or what the difference is between the table in `systemd.unit.5` and the podman one. Giving up for now, waiting for inspiration.

Reply:
I give up. Man page highlights are inconsistent. The following groff:
```groff
.SH DESCRIPTION
.PP
Monitor and print events that occur in Podman. Each event includes a timestamp,
a type, a status, name (if applicable), and image (if applicable).  The default logging
mechanism is \fIjournald\fP\&. This can be changed in containers.conf by changing the \fB\fCevents_logger\fR
value to \fB\fCfile\fR\&.  Only \fB\fCfile\fR and \fB\fCjournald\fR are accepted. A \fB\fCnone\fR logger is also
available, but this logging mechanism completely disables events; nothing is reported by
\fB\fCpodman events\fR\&.

.PP
By default, streaming mode is used, printing new events as they occur.  Previous events can be listed via \fB\fC--since\fR and \fB\fC--until\fR\&.
```
...produces the following display: 
![man](https://github.com/containers/podman/assets/265639/30d90277-4f9f-4007-8880-f1c67b87317e)

Note the proliferation of `\f<whatever>` in the source. The only ones that actually show up in the terminal view are the ones with `--` (the options). This is too obscure for me.